blocks/hero-block/features/hero/default.jsx

```diff
@@ -0,0 +1,166 @@
+import React, { Fragment } from "react";
+import PropTypes from "@arc-fusion/prop-types";
+
+// Arc Themes Components - Base set of components used to compose blocks
+// https://github.com/WPMedia/arc-themes-components/
+import { Button, Heading, HeadingSection, Paragraph, Stack } from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-hero";
+
+function Hero({ customFields }) {
+	const {
+		layout = "overlay",
+		alignment = "center",
+		description,
+		headline,
+		imageURLDesktop,
+		imageURLMobile,
+		desktopMediaBreakpoint,
+		imageAltText,
+		link1Action,
+		link1Text,
+		link1Type,
+		link2Action,
+		link2Text,
+		link2Type,
+		subHeadline,
+		variant = "dark",
+	} = customFields;
+
+	const classes = [
+		BLOCK_CLASS_NAME,
+		`${BLOCK_CLASS_NAME}--${layout}`,
+		variant === "dark" ? `${BLOCK_CLASS_NAME}--dark` : `${BLOCK_CLASS_NAME}--light`,
+	].join(" ");
+
+	const HeadingWrapper = headline ? HeadingSection : Fragment;
+
+	return (
+		<div className={classes}>
+			<picture>
+				<source srcSet={imageURLDesktop} media={`(${desktopMediaBreakpoint})`} />
```

Comment:
I think this should be removed and we pick either the 16:9 or the 3:4 image as our MVP.  We can discuss this on the call, but:
* I think it sets a precedent that we don't want to continue (being asked to do this for any image)
* I don't think it's scalable (having to add potentially more breakpoints later)
* I think the conversation should be moved to the "responsive/scalable iamge" component that we have yet to develop.

I think we should keep this specific use case in mind for the responsive images, but for MVP... I don't think this feature should be implemented at this time.

Reply:
this has been addressed to some extent. The image could take in an aspect ratio 16:9 or 9:16. But this is still up for work